kcidb/monitor/spool/__init__.py

```diff
@@ -97,6 +97,12 @@ def post(self, notification, timestamp=None):
             timestamp:      An "aware" datetime.datetime object specifying the
                             notification creation time, or None to use
                             datetime.datetime.now(datetime.timezone.utc).
+            time_to_send:   An "aware" datetime.datetime object specifying the
+                            time notification should be sent out,
+                            or None to use
+                            datetime.datetime.now(datetime.timezone.utc)
+                            + datetime.timedelta(hours=1) .
```

Comment:
We shouldn't be putting the default due delay here, in this function. For most of notifications it won't be suitable. Instead, the default here should be "due immediately",  which should be the creation time, i.e. the `timestamp` argument. Then subscriptions should be able to request their own due delay.